testing/kglobalaccel/APKBUILD

```diff
@@ -7,11 +7,12 @@ pkgdesc="Add support for global workspace shortcuts"
 arch="all"
 url="https://community.kde.org/Frameworks"
 license="LGPL-2.1-or-later"
-depends_dev="qt5-qtx11extras-dev kconfig-dev kcoreaddons-dev kcrash-dev kdbusaddons-dev kwindowsystem-dev"
+depends_dev="qt5-qtx11extras-dev kconfig-dev kcoreaddons-dev kcrash-dev
+	kdbusaddons-dev kwindowsystem-dev"
```

Comment:
I personally prefer all dependencies on a single line, my editor will break the lines internally.